### ISSUE TITLE
Schedule backups

### ISSUE THREAD

**Issue by felipecruz91** [docker/volumes-backup-extension#98]
**Is your feature request related to a problem? Please describe.**
Schedule a backup, say every day or week.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by peschee** [comment]
This would be a very nice addition!

**Comment by cohen87** [comment]
Looking forward to this enhancement!!

**Comment by iftaras** [comment]
I concur! This would be an awesome feature!

**Comment by klodoma** [comment]
I think as a start, it would be enough if we could just execute the creation of the backup in a command line or smth like that.

**Comment by chrisk-7777** [comment]
> I think as a start, it would be enough if we could just execute the creation of the backup in a command line or smth like that.

I strongly agree, a scheduler is nice, but opening it up to the CLI would facilitate much more flexibility from user land (including custom scheduling).

**Comment by V3semir** [comment]
This, or at least the ability to backup multiple volumes at once.

**Comment by kevingt3** [comment]
I'm really interested for this feature of scheduled backup
